test/extended/oauth/oauthcertfallback.go

```diff
@@ -32,6 +33,12 @@ var _ = g.Describe("[Feature:OAuthServer] OAuth server", func() {
 	oc := exutil.NewCLI("oauth", exutil.KubeConfigPath())
 
 	g.It("has the correct token and certificate fallback semantics", func() {
+		// Authenticator order has changed to match kubernetes as of
+		// 4.4, which precludes running this test against a skewed
+		// cluster.
+		if len(os.Getenv("TEST_UNSUPPORTED_ALLOW_VERSION_SKEW")) > 0 {
+			e2e.Skipf("Test is disabled to allow cluster components to have different versions")
```

Comment:
Could probably replace this message with the comment (and remove the comment) -

Reply:
Done